tlcc/tlcc_test.go

```diff
@@ -9,49 +9,72 @@ package tlcc
 import (
```

Comment:
I feel like these changes should be separate unless they fail linting.

Reply:
(Referring to Morgan's comment which i kindof agree) I guess that also applies to above change to `tlcc/Makefile` ..

Reply:
well ... all the changes I made in this PR are to fix linting

Reply:
hmm, lint usually shouldn't force you to change the behaviour ;-?  At least the test seems to be changed beyond what i would have expected to just make lint work ...  Now i'm not religious of not piggibacking some small change on a PR, but would be good if you say something about the changed/added behaviour?

Reply:
I agree that tlcc test fixing is not small anymore. Will create a separate PR for this once linter is enabled again.